src/scene/materials/standard-material-parameters.js

```diff
@@ -172,12 +182,58 @@ const standardMaterialParameterTypes = {
 
     reflectivity: 'number',
     refraction: 'number',
+    refractionMap: 'texture',
+    refractionMapChannel: 'string',
+    refractionMapUv: 'number',
+    refractionMapTiling: 'vec2',
+    refractionMapOffset: 'vec2',
+    refractionMapMapRotation: 'number',
+    refractionTint: 'boolean',
+    refractionVertexColor: 'boolean',
+    refractionVertexColorChannel: 'string',
```

Comment:
Seems like there's an opportunity to define map properties in a function perhaps? Most maps share the same set.

Reply:
Yes, that would be the third such function :D 

But I totally agree, this is annoying, repetitive and easy to miss, which is made abundantly clear by this PR.

Reply:
I wrote a function for this and condensed these declarations to a simple function call.